Fluorescent Reporter Proteins/ky021423.gb

```diff
@@ -0,0 +1,22 @@
+LOCUS       mScarlet:_KY021423.1     702 bp ds-DNA     linear       09-AUG-2021
```

Comment:
You can remove this file and also ky484012.gb: it looks like they are both NCBI accessions, which have been automatically downloaded into the `GenBank_imports.gb` file.

Reply:
Ah this was one of the issues I was concerned about, so it's good for me to fix it! 

I can delete KY021423.gb because it is as specified, but the sequence for KY021423 in GenBank lacked stop codon(s) so they were added in the file I included. The other sequence files as well are derived from Addgene but have edits to remove extraneous DNA parts. So it sounds like I should not be using the DATA source fields in cases like this (even for posterity) and should instead clear those fields, and use either unrelated filenames or another naming convention.

Reply:
I would say that if the sequence isn't identical, then put it in the design notes instead of in the source field.

Reply:
It's like if I said: "Let's use E0040, but we'll codon optimize it for B. subtilis" then it's not actually E0040 any more, and labelling it as E0040 will only create confusion.